samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/StopSignDetection.csproj

```diff
@@ -13,6 +13,14 @@
     <ProjectReference Include="..\StopSignDetectionML.Model\StopSignDetectionML.Model.csproj" />
   </ItemGroup>
 
+  <Target Name="DownloadModelFiles" BeforeTargets="Build">
+    <DownloadFile 
+      SourceUrl="https://aka.ms/mlnet-object-detection-tutorial-model"
```

Comment:
Neat, it chooses the right file name for this?

Reply:
Yep! At least it built and ran for me when I tested it

Reply:
That aka.ms links directly to MLModel.zip. I believe as long as that is the model file name it will work fine.

Reply:
This was mostly for my info, I haven't used DownloadFile task a lot.  Looks nifty for this sort of thing in a sample.